packages/server/lib/request.js

```diff
@@ -662,14 +662,18 @@ module.exports = function (options = {}) {
       })
     },
 
-    sendPromise (userAgent, automationFn, options = {}) {
+    async sendPromise (userAgent, automationFn, getFixture, options = {}) {
       _.defaults(options, {
         headers: {},
         gzip: true,
         cookies: true,
         followRedirect: true,
       })
 
+      if (options.fixture) {
+        options.form = await getFixture(options.fixture.filePath, options.fixture.encoding)
```

Comment:
My original suggestion was a bit naive with the amount of messaging we are doing in the `cy.request()` command with assigning headers, validating the body structure etc. Your original approach was definitely better than this suggestion. 

What you have here makes assumptions that the fixture data that's for a form, where someone may want it for any body type.

Using the `cy.fixture()` command will cache the fixture data for the duration fo the spec, which we won't want to introduce, but we can call into the backend directly. something like?
```js
// request.ts
Promise.try(() => {
  if (!options.fixture) return
  return Cypress.backend('get:fixture', options.fixture.filePath, options.fixture.filePath.encoding)`. 
}).then((fixtureData) => {
    options.body = fixtureData
    // handle fixture data
    // handle blob data
}).then(() => {
   return Cypress.backend('http:request', requestOpts)
})
....
```

I was really hoping to avoid an extra client-side->backend-> exchange with the introduction of this feature. 

@chrisbreiding maybe you have some ideas on keeping this a single exchange? maybe its not necessary?